### PR TITLE
Feat/new workflow: Common Import Credentials

### PR DESCRIPTION
**Pull request by rhofvendahl**
This PR creates a new workflow `Common Import Credentials`, describing a simple interaction between an importer, broker and CBP involving some common credentials.

### CODE REVIEW

**Comment by TallTed** [comment]
Could the title (and subsequent) of this workflow be changed from `Common Import Credentials` to `Common Credentials for Imports`? There might be another rephrasing that would be better. I'm just concerned with making it read more clearly, especially for non-English and non-Importer consumers who are just trying to learn.

Relatedly, please change the title of this PR to [`Feat/new workflow: Common Credentials for Imports`](https://github.com/w3c-ccg/traceability-vocab/pull/853#top) so it's clearly different from other future/past `Feat/new workflow` PRs.

**Comment by rhofvendahl** [comment]
@TallTed the intention of the title was to describe these credentials as common constituents of an importation workflow. I was thinking the title suggests this in a succinct way, but I'd appreciate an outside view so if you feel strongly I can change it. Regardless, I've updated the description to reword things a bit - hopefully that helps some!

File: docs/sections/workflows.html [hunk 1]
```diff
@@ -134,6 +134,65 @@ <h3>CBP Steel Tech Demo 2023</h3>
...
+        <li><a href="https://w3id.org/traceability/#CommercialInvoiceCredential">Commercial Invoice Credential</a></li><li><a href="https://w3id.org/traceability/#CTPATCertificate">CTPAT Certificate</a></li><li><a href="https://w3id.org/traceability/#IntentToImportCredential">Intent To Import Credential</a></li><li><a href="https://w3id.org/traceability/#MultiModalBillOfLadingCredential">Multimodal Bill Of Lading Credential</a></li><li><a href="https://w3id.org/traceability/#PurchaseOrderCredential">Purchase Order Credential</a></li>
+      </ol>
+      <pre class='example yml'>id: https://w3id.org/traceability/#common-import-credentials
+title: Common Import Credentials
+description: >-
+    Common Import Credentials
+tags: 
+  - Steel
+  - Agriculture
+  - OilAndGas
```
Comment (mkhraisha) [inline docs/sections/workflows.html:164]: Multimodal BOL, Commercial Invoice don't apply to O&G, maybe remove OG?

**Review by mkhraisha — CHANGES_REQUESTED** [review]

File: docs/credentials-with-issuer-dependent-terms.json [hunk 2]
```diff
@@ -89,7 +89,7 @@
   },
   {
     "type": "MillTestReportCredential",
-    "count": 187
+    "count": 191
```
Comment (OR13) [inline docs/credentials-with-issuer-dependent-terms.json:92]: @nissimsan just a warning about this number going up : /

**Review by OR13 — APPROVED** [review]

File: docs/credentials-with-issuer-dependent-terms.json (hunk 2 above)
Comment (nissimsan) [reply docs/credentials-with-issuer-dependent-terms.json:92]: Oh I know... Looking at it all day. Very sad! :(

**Comment by mkhraisha** [comment]
Happy to merge out of band if name change goes through as ted requested and the OG comment is addressed

**Comment by rhofvendahl** [comment]
@mkhraisha I've improved the description in response to Ted's suggestion (see above), and made the requested changes. Ok to merge?

**Comment by nissimsan** [comment]
@rhofvendahl , are you aware of https://w3c-ccg.github.io/traceability-vocab/#us-cbp-entry?

**Review by mkhraisha — APPROVED** [review]

**Pull request merged**